src/main/java/core/basesyntax/service/FruitTransactionService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitTransactionService {
+    Map<String,Integer> addOrUpdate(List<FruitTransaction> fruitTransactions);
```

Comment:
1) Why did you decide to return a map?
2) Since you've decided to do so, why don't you use the returned map?